src/fragments/attributes.ts

```diff
@@ -14,14 +14,21 @@ export const attributeValueFragment = gql`
       ...FileFragment
     }
     reference
-    richText
     boolean
     date
     dateTime
     value
   }
 `;
 
+export const attributeValueDetailsFragment = gql`
+  ${attributeValueFragment}
+  fragment AttributeValueDetailsFragment on AttributeValue {
+    ...AttributeValueFragment
+    richText
```

Comment:
Use the new fragment you created instead ‼️

Reply:
It's beginning of march and I think I already won the 🏺 of the month award